### PR TITLE
Regis/proofread 20170725

### PR DESCRIPTION
**Pull request by regisb**
J'ai pas vraiment réussi à améliorer le code de l'admin. Par contre j'ai réparé un bug dans les modèles.

### CODE REVIEW

File: disquaire_project/manager/models.py [hunk 1]
```diff
@@ -20,11 +20,13 @@ def __str__(self):
         return self.name
 
 class Disk(models.Model):
+    # There is no unicity constraint and no index on the reference. Are you sure you need this?
```
Comment (celine-m-s) [inline disquaire_project/manager/models.py:23]: I aggree.
We need to add constraints and index. Is it something that has to go on the views or on the model? I though I should to a validation to check the reference is an integer and is unique.

File: disquaire_project/manager/models.py [hunk 2]
```diff
@@ -20,11 +20,13 @@ def __str__(self):
...
     reference = models.IntegerField('référence')
     created_at = models.DateTimeField('Date de création', auto_now_add=True)
+    # what does "active" mean?
```
Comment (celine-m-s) [inline disquaire_project/manager/models.py:26]: It means it should appear on the front office as "available". A user can book an activated disk only.

File: disquaire_project/manager/models.py (hunk 1 above)
Comment (regisb) [reply disquaire_project/manager/models.py:23]: Unicity constraints go to the model. Then, in the views, on form submission, for validation will automatically raise an error when unicity constraints are violated.

File: disquaire_project/manager/models.py (hunk 2 above)
Comment (regisb) [reply disquaire_project/manager/models.py:26]: So, maybe this field should be renamed to "available"?

File: disquaire_project/manager/models.py (hunk 1 above)
Comment (celine-m-s) [reply disquaire_project/manager/models.py:23]: 👍 
Will do it.

File: disquaire_project/manager/models.py (hunk 2 above)
Comment (celine-m-s) [reply disquaire_project/manager/models.py:26]: Done

**Pull request merged**